app/react/Settings/components/CollectionSettings.js

```diff
@@ -269,11 +283,27 @@ export class CollectionSettings extends Component {
             </div>
             <div className="form-group">
               <label className="form-group-label" htmlFor="mapTilerKey">
-                {t('System', 'MapTiler key')}
+                <Translate>MapTiler key</Translate>
               </label>
               <Control.text model=".mapTilerKey" className="form-control" />
+              <div className="alert alert-info">
+                <div className="force-ltr">You can use your own MapTiler API key</div>
+              </div>
+            </div>
+            <div className="form-group">
+              <label className="form-group-label" htmlFor="mapStartingPoint">
+                <Translate>Map starting point</Translate>
+              </label>
+              <Geolocation model=".mapStartingPoint" />
+              <div className="alert alert-info">
+                <div className="force-ltr">
+                  You can set the default starting point for your geolocation properties.
```

Comment:
@konzz I've added an info message about the options for the maps, please can you take a look about the correctness of it.
As #3066 is asking about to link settings with documentation, maybe it's required to update this link with these new options [Managing Settings](https://uwazi.readthedocs.io/en/initial-setup/admin-docs/managing-settings.html?highlight=PRIVATE%20INSTANCE#how-to-edit-your-site-settings)